slowest.php

```diff
@@ -29,7 +29,7 @@
 require_once('../../../config.php');
 require_once($CFG->libdir.'/adminlib.php');
 
-$script = optional_param('script', '', PARAM_LOCALURL);
+$script = optional_param('script', '', PARAM_RAW);
```

Comment:
can you add a comment here saying why we need raw over url param please

Reply:
I have put in a comment as requested. It is need because the script name may be a task name which does not follow the URL pattern.